lib/internal/Magento/Framework/Url/Validator.php

```diff
@@ -45,11 +47,14 @@ public function isValid($value)
     {
         $this->_setValue($value);
 
-        if (!\Zend_Uri::check($value)) {
-            $this->_error(self::INVALID_URL);
-            return false;
-        }
+        try {
```

Comment:
Rather than using `Zend\Uri\UriFactory` here, this logic can be replaced by using `Zend\Validator\Uri`